.vitepress/config.ts

```diff
@@ -33,60 +33,19 @@ export default defineConfig({
       text: 'Edit this page',
     },
     nav: [
-      { text: 'Guides', link: data.userGuideLinks[0].link },
-      { text: 'Codeflow', link: data.codeflow[0].link },
-      { text: 'API', link: data.apiLinks[0].link },
-      { text: 'WebContainers', link: data.webcontainersLinks[0].link },
-      { text: 'Enterprise', link: data.enterpriseLinks[0].link! },
+      { text: 'Guides', link: defaultGroupLink('userGuide') },
+      { text: 'Codeflow', link: defaultGroupLink('codeflow') },
+      { text: 'API', link: defaultGroupLink('api') },
+      { text: 'WebContainers', link: defaultGroupLink('webcontainers') },
+      { text: 'Enterprise', link: defaultGroupLink('enterprise') },
     ],
     sidebar: {
-      '/guides/': [
-        {
-          text: 'User Guide',
-          collapsible: true,
-          collapsed: false,
-          items: data.userGuideLinks,
-        },
-        {
-          text: 'Integration Guide',
-          collapsible: true,
-          collapsed: false,
-          items: data.integrationGuideLinks,
-        },
-      ],
-      '/codeflow/': [
-        {
-          text: 'Codeflow',
-          collapsible: false,
-          collapsed: false,
-          items: data.codeflow,
-        },
-      ],
-      '/platform/api/': [
-        {
-          text: 'StackBlitz API',
-          collapsible: true,
-          collapsed: false,
-          items: data.apiLinks,
-        },
-      ],
-      '/platform/webcontainers/': [
-        {
-          text: 'WebContainers',
-          collapsible: true,
-          collapsed: false,
-          items: data.webcontainersLinks,
-        },
-      ],
-      '/enterprise/': [
-        {
-          text: 'Enterprise Edition',
-          collapsible: false,
-          collapsed: false,
-          items: data.enterpriseLinks,
-        },
-      ],
-    },
+      '/guides/': linkGroup(['userGuide', 'integrationGuide']),
```

Comment:
Always include all link groups in sidebar, and choose which are collapsed based on route.